VTEX - VTEX ID API.json

```diff
@@ -72,6 +72,78 @@
                 },
                 "deprecated": false
             }
+        },
+        "/{accountName}.{environment}.com.br/vtexid/audience/webstore/provider/oauth/exchange": {
+            "post": {
+                "tags": ["Authentication"],
+                "summary": "Exchange OAuth access token for VTEX credential",
+                "description": "If your store uses a headless frontend, you can authenticate users via OAuth and use this endpoint to exchange the OAuth access token for a VTEX authentication token. This VTEX token can be used to authenticate your frontend requests to VTEX APIs.\n\r\n\rLearn more about how to implement [headless authentication](https://developers.vtex.com/docs/guides/headless-authentication) in your store.",
```

Comment:
Link quebrado

Reply:
Esse guia está em produção e vou abrir PR nos próximos dias. 

Além disso, a feature está em alpha, a prioridade é enviar a referência de API para os clientes. E aí quando for pra publicar, a API ref e o guia estarão disponíveis.

Reply:
Beleza!